src/vs/workbench/contrib/interactive/browser/interactiveEditor.ts

```diff
@@ -149,7 +149,7 @@ export class InteractiveEditor extends EditorPane {
 		this.#notebookExecutionStateService = notebookExecutionStateService;
 		this.#extensionService = extensionService;
 
-		this.#notebookOptions = new NotebookOptions(configurationService, notebookExecutionStateService, { cellToolbarInteraction: 'hover', globalToolbar: true, defaultCellCollapseConfig: { codeCell: { inputCollapsed: true } } });
```

Comment:
Do we still need `defaultCellCollapseConfig` though?

Reply:
it was not used anywhere, and the code cells were still collapsed when testing this change

Reply:
Oh, ok. I'll see if something needs to be cleaned up there